packages/@jsii/integ-test/test/build-cdk.test.ts

```diff
@@ -27,62 +36,98 @@ describe('Build CDK', () => {
+        owner: 'aws',
+        repo: 'aws-cdk'
+      });
 
-    // download and extract code
-    const code = await downloadReleaseAsset(`https://api.github.com/repos/aws/aws-cdk/tarball/${release.data.tag_name}`);
-    const srcDir = path.join(buildDir, `aws-aws-cdk-${release.data.target_commitish.substring(0, 7)}`);
-    await extractFileStream(code, buildDir);
+      // download and extract code
+      const code = await downloadReleaseAsset(
+        `https://api.github.com/repos/aws/aws-cdk/tarball/${release.data.tag_name}`
+      );
+      const srcDir = path.join(
+        buildDir,
+        `aws-aws-cdk-${release.data.target_commitish.substring(0, 7)}`
+      );
+      await extractFileStream(code, buildDir);
 
-    // install cdk dependencies
-    await processes.spawn('yarn', ['install', '--frozen-lockfile'], {
-      cwd: srcDir
-    });
+      // install cdk dependencies
+      await processes.spawn('yarn', ['install', '--frozen-lockfile'], {
+        cwd: srcDir
+      });
 
-    // build cdk build tools
-    await processes.spawn('npx', [
-      'lerna',
-      '--stream',
-      '--scope', 'cdk-build-tools',
-      '--scope', 'pkglint',
-      '--scope', 'awslint',
-      'run', 'build',
-    ], { cwd: srcDir });
+      // build cdk build tools
+      await processes.spawn(
+        'npx',
+        [
+          'lerna',
+          '--stream',
+          '--scope',
+          'cdk-build-tools',
+          '--scope',
+          'pkglint',
+          '--scope',
+          'awslint',
+          'run',
+          'build'
+        ],
+        { cwd: srcDir }
+      );
 
-    // build jsii modules
-    await processes.spawn('npx', [
-      'lerna',
-      '--stream',
-      '--scope', '@aws-cdk/*',
-      '--scope', 'aws-cdk',
-      'run', 'build',
-      '--', '--jsii', path.join(JSII_DIR, 'bin', 'jsii'),
-    ], { cwd: srcDir });
+      // build jsii modules
+      await processes.spawn(
+        'npx',
+        [
+          'lerna',
+          '--stream',
+          '--scope',
+          '@aws-cdk/*',
+          '--scope',
+          'aws-cdk',
+          'run',
+          'build',
+          '--',
+          '--jsii',
+          path.join(JSII_DIR, 'bin', 'jsii')
+        ],
+        { cwd: srcDir }
+      );
 
-    // build the rest
-    await processes.spawn('npx', [
-      'lerna',
-      '--stream',
-      '--ignore', '@aws-cdk/*',
-      '--ignore', 'aws-cdk',
-      '--ignore', 'cdk-build-tools',
-      '--ignore', 'pkglint',
-      '--ignore', 'awslint',
-      'run', 'build',
-    ], { cwd: srcDir });
+      // build the rest
+      await processes.spawn(
+        'npx',
+        [
+          'lerna',
+          '--stream',
+          '--ignore',
+          '@aws-cdk/*',
+          '--ignore',
+          'aws-cdk',
+          '--ignore',
+          'cdk-build-tools',
+          '--ignore',
+          'pkglint',
+          '--ignore',
+          'awslint',
+          'run',
+          'build'
+        ],
```

Comment:
I don't like this change :/ It hurts readability :/

Reply:
I agree. Too many newlines.